internal/persistence/sql/migrations/sql/20210914134624000000_legacy-cleanup.cockroach.up.sql

```diff
@@ -0,0 +1 @@
+DROP TABLE "keto_namespace";
```

Comment:
This table was actually never used, not even in the old versions of Keto. Therefore just dropping it.